Sources/SectionScrubber.swift

```diff
@@ -76,6 +98,10 @@ public class SectionScrubber: UIView {
             if let scrubberImage = self.scrubberImage {
                 self.scrubberWidth = scrubberImage.size.width
                 self.scrubberImageView.image = scrubberImage
+                // scrubber is hidden by default
```

Comment:
Scrubber is hidden by default doesn't express anything I don't know. I can just look at the gif and know that this is how it works,